gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/upgrade/DefaultInstallationUpgrader.java

```diff
@@ -1,50 +0,0 @@
-/**
```

Comment:
Did you delete this file because Installation is already initialized in another Upgrader ?